qa/rpc-tests/test_framework/util.py

```diff
@@ -184,7 +184,8 @@ def initialize_datadir(dirname, n,bitcoinConfDict=None,wallet=None):
         os.makedirs(datadir)
     
     defaults = {"server":1, "discover":0, "regtest":1,"rpcuser":"rt","rpcpassword":"rt",
-                "port":p2p_port(n),"rpcport":str(rpc_port(n)),"listenonion":0,"maxlimitertxfee":0}
+                "port":p2p_port(n),"rpcport":str(rpc_port(n)),"listenonion":0,"maxlimitertxfee":0, "debug":"net",
+                "debug":"parallel", "debug":"thin", "debug":"blk", "debug":"req", "debug":"mempool"}
```

Comment:
One KV pair per line would be much more readable. We'd also get an history that is much easier to understand as it'll be easy to see who changed what parameter, and in what context instead of seeing a bulk change of all parameters.